### PR TITLE
Update ucs.js

### PR DESCRIPTION
**Pull request by Muthuwebworm**
Fix for GAPI-24103 - Search in Contact Directory shouldn't be case sensitive

### CODE REVIEW

File: src/service/controllers/ucs.js [hunk 1]
```diff
@@ -112,18 +112,19 @@ searchContacts = (req, res) => {
 	if (searchedTerm.indexOf('\\') === 0) {
 		searchedTerm = searchedTerm.substring(1);
 	}
+	searchedTerm = new RegExp(searchedTerm, 'i');
```
Comment (tichon29) [inline src/service/controllers/ucs.js:115]: I guess this update can generate issues.
For example, if someone was looking for a string with some special characters from a regexp (like . or ? or ...), the behavior will be updated.
Why not simply do
searchedTerm = searchedTerm.toLowerCase();

And, then compare with with lowercase version of emailaddress, firstname or lastname (or companyname)

Comment (sblecon) [reply src/service/controllers/ucs.js:115]: Agree with Stéphane, no regex should be used for this case.
Proposal from Stéphane H sounds the good one.

Comment (Muthuwebworm) [reply src/service/controllers/ucs.js]: Agreed with you Stephane H. Please refer to the new Pull Request

**Review by sblecon — APPROVED** [review]
Fine with latest update.

**Review by tichon29 — APPROVED** [review]

**Pull request merged**